src/main/java/io/split/android/client/SplitFactoryHelper.java

```diff
@@ -99,13 +99,20 @@ String getDatabaseName(SplitClientConfig config, String apiToken, Context contex
     }
 
     private String buildDatabaseName(SplitClientConfig config, String apiToken) {
-        int apiTokenLength = apiToken.length();
+        if (apiToken == null) {
```

Comment:
I think the SDK keys is validated previously in Input Validation somewhere, so this check wouldn't be necessary.